src/main/java/banking/BankingSystem.java

```diff
@@ -0,0 +1,171 @@
+package banking;
+
+import banking.dao.AccountDAO;
+import banking.model.Account;
+import banking.model.Card;
+
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+
+public class BankingSystem {
+
+    private Account currentAccount;
+    private int amountSend;
+    private boolean isLogin;
+    private final CardGenerator cardGenerator;
+    private final CardValidator cardValidator;
+    private final AccountDAO accountDAO;
+    private final Scanner scanner;
+
+    public BankingSystem(AccountDAO accountDAO) {
+        cardGenerator = new CardGenerator();
```

Comment:
Can you pass `cardValidator` into `CardGenerator` constructor?

Reply:
Just done